packages/dtslint/test/index.test.ts

```diff
@@ -14,98 +14,136 @@ describe("dtslint", () => {
     noEmit: true,
     forceConsistentCasingInFileNames: true,
   };
+  function based(extra: object) {
```

Comment:
what a function name

Reply:
I just write whatever bubbles out of my subconscious.